BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -227,6 +227,14 @@ extension StoresTableViewController {
         
     }
     
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        
+        let store = stores[indexPath.row]
```

Comment:
Nice. Add a guard to make sure that :
1. There are stores
2. The selected cell was a StoreTableViewCell
3. The index is not out of bounds.